ember-async-data/type-tests/load-test.ts

```diff
@@ -1,23 +1,17 @@
 import TrackedAsyncData from '../src/tracked-async-data';
 import loadHelper, { load } from '../src/helpers/load';
 import { expectTypeOf } from 'expect-type';
-import {
-  EmptyObject,
-  FunctionBasedHelperInstance,
-} from '@ember/component/helper';
 
 expectTypeOf(load).toEqualTypeOf<
   <T>(data: T | Promise<T>) => TrackedAsyncData<T>
 >();
 
 expectTypeOf(load(true)).toEqualTypeOf(new TrackedAsyncData(true));
 
-type LoadHelper = abstract new <T>() => FunctionBasedHelperInstance<{
-  Args: {
-    Positional: [T | Promise<T>];
-    Named: EmptyObject;
-  };
-  Return: TrackedAsyncData<T>;
-}>;
+expectTypeOf<ReturnType<typeof loadHelper<number>>>().toEqualTypeOf<
+  TrackedAsyncData<number>
+>();
 
-expectTypeOf(loadHelper).toEqualTypeOf<LoadHelper>();
+expectTypeOf<ReturnType<typeof loadHelper<string>>>().toEqualTypeOf<
+  TrackedAsyncData<string>
+>();
```

Comment:
`expect-type` has a type utility for this, though we already cover this above.

```suggestion
expectTypeOf(load("hello")).returns.toEqualTypeOf<TrackedAsyncData<string>>();
```